keps/sig-network/2438-dual-stack-apiserver/kep.yaml

```diff
@@ -14,8 +14,6 @@ reviewers:
 approvers:
   - "@thockin"
   - TBD
```

Comment:
@khenidak since your fingerprints are all over dual-stack, I think you should be an approver here :)